src/App.js

```diff
@@ -1,25 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Home from './routes/Home';
+import Categories from './routes/Categories';
+import CategoryDetail from './routes/CategoryDetail';
+import PokemonDetail from './routes/PokemonDetail';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <Routes>
+    <Route path="/" element={<Home />}>
+      <Route path="/" element={<Categories />} />
+      <Route path="/type/:typeId/:typeName" element={<CategoryDetail />} />
+      <Route path="pokemon/:pokemonId" element={<PokemonDetail />} />
+    </Route>
+  </Routes>
+);
```

Comment:
- [ ] Good job creating a beautful design for your projet that follows the typogra[hy and the layout of the the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) , however there is still something you can improve, Kindly make that in the mobile version the content of the page fill all the page according the design template 

| Your design | The template design |
| :---: | :---: | 
| ![Capture](https://user-images.githubusercontent.com/80458199/233452864-15244778-b337-4bac-a1aa-ad0fd7b2a46b.PNG) | ![Capture 2](https://user-images.githubusercontent.com/80458199/233452885-fc31cf06-419b-4d30-b5a5-f21dcfad1e11.PNG) |